packages/runtime/container-runtime/src/test/types/validateContainerRuntimePrevious.ts

```diff
@@ -795,6 +795,7 @@ declare function get_old_InterfaceDeclaration_IProvideSummarizer():
 declare function use_current_InterfaceDeclaration_IProvideSummarizer(
     use: TypeOnly<current.IProvideSummarizer>);
 use_current_InterfaceDeclaration_IProvideSummarizer(
+    // @ts-expect-error compatibility expected to be broken
```

Comment:
there need to go in the package.json, and then you need to run `npm run typetests:gen`. if they are not in the package json they will be lost on the next generation

Reply:
This is what it will add to your package.json
       "ClassDeclaration_Summarizer": {
          "forwardCompat": false
        },
        "InterfaceDeclaration_IProvideSummarizer": {
          "forwardCompat": false
        },
        "InterfaceDeclaration_ISummarizer": {
          "forwardCompat": false
        }